mopidy_pidi/frontend.py

```diff
@@ -118,6 +118,16 @@ def update_track(self, track, time_position=None):
                 length=float(track.length)
             )
 
+        art = None
+        track_images = self.core.library.get_images([track.uri]).get()
+        if track.uri in track_images:
+            track_images = track_images[track.uri]
+            for image in track_images:
+                if image.height == 300 and image.width == 300:
```

Comment:
Probably incompatible with any backend other than Spotify, since it now *requires* an album art image be 300x300. We should scan through and find the first image that's > 240x240. Perhaps failing over to the largest available where that doesn't work out?